src/pyodide-py/_pyodide/docstring.py

```diff
@@ -0,0 +1,31 @@
+from textwrap import dedent
+
+
+def dedent_docstring(docstring):
```

Comment:
It might help to have a doctest to have an example what this function does.

Reply:
Okay I added a bit of explanation and a doctest.